GoogleDataTransport.podspec

```diff
@@ -92,7 +93,7 @@ Shared library for iOS SDK data transport needs.
     test_spec.scheme = { :code_coverage => true }
     test_spec.platforms = {:ios => ios_deployment_target, :osx => osx_deployment_target, :tvos => tvos_deployment_target}
     test_spec.requires_app_host = false
-    test_spec.source_files = ['GoogleDataTransport/GDTCORTests/Unit/**/*.{h,m}'] + common_test_sources
+    test_spec.source_files = ['GoogleDataTransport/GDTCORTests/Unit/**/*.{h,m}'] + common_test_sources + common_cct_test_sources
```

Comment:
Should the CCT-Tests-Unit target be deleted if all the tests are here now?

Reply:
Good question. Currently the unit tests are still in separate targets, but they share test utils.